presto-main/src/main/java/io/prestosql/sql/planner/PlanOptimizers.java

```diff
@@ -566,6 +567,11 @@ public PlanOptimizers(
                 new IndexJoinOptimizer(metadata), // Run this after projections and filters have been fully simplified and pushed down
                 new LimitPushDown(), // Run LimitPushDown before WindowFilterPushDown
                 new WindowFilterPushDown(metadata), // This must run after PredicatePushDown and LimitPushDown so that it squashes any successive filter nodes and limits
+                new IterativeOptimizer(
```

Comment:
Actually, it does not need to be in a separate IterativeOptimizer. I did it this way so that another upcoming change could be easily rebased (https://github.com/prestosql/presto/pull/4242). However, that change adds a bunch of rules together with a session property to use them or not. The two rules introduced in this PR must fire regardless of that toggle so they mustn't be in the same Optimizer.
Instead of adding a new IterativeOptimizer, I added the new rules to an existing one.